packages/labs/webpack/src/cli.ts

```diff
@@ -41,10 +41,14 @@ function main(config: webpack.Configuration): 0|1 {
   return exitCode;
 }
 
-if (require.main === module) {
+export function main() {
```

Comment:
why this refactoring? I'd rather not have process.exitCode happening if main() is called from a test

Reply:
Artifact of dev before `cli.ts` was a valid entry_point. Will revert. Thanks